src/detectors/ScanningStrategyDetector.spec.ts

```diff
@@ -249,6 +250,46 @@ describe('ScanningStrategyDetector', () => {
       });
     });
 
+    it('remote private GitLab without protocol in the URL', async () => {
+      const repoPath = 'gitlab.com/DXHeroes/dx-scanner.git';
+      const nock = new GitLabNock('DXHeroes', 'dx-scanner', 'gitlab.com');
+      nock.listProjects();
+      nock.listGroups();
+      nock.checkVersion();
+      nock.getRepoResponse(200, { visibility: 'private' });
+
+      const scanningStrategyDetector = await createScanningStrategyDetector({ uri: repoPath, auth: 'valid_token' });
+      const result = await scanningStrategyDetector.detect();
+
+      expect(result).toEqual({
+        accessType: AccessType.private,
+        localPath: undefined,
+        remoteUrl: `https://${repoPath}`,
+        isOnline: true,
+        serviceType: ServiceType.gitlab,
+      });
+    });
+
+    it('remote private GitLab without protocol in the URL', async () => {
+      const repoPath = 'gitlab.com/DXHeroes/dx-scanner.git';
+      const nock = new GitLabNock('DXHeroes', 'dx-scanner', 'gitlab.com');
+      nock.listProjects();
+      nock.listGroups();
+      nock.checkVersion().reply(403);
+      nock.getRepoResponse(403);
+
+      const scanningStrategyDetector = await createScanningStrategyDetector({ uri: repoPath, auth: 'valid_token' });
```

Comment:
As it returns 403, it's not valid token

```suggestion
      const scanningStrategyDetector = await createScanningStrategyDetector({ uri: repoPath, auth: 'invalid_token' });
```